src/engraving/libmscore/accidental.cpp

```diff
@@ -559,6 +563,9 @@ EngravingItem* Accidental::drop(EditData& data)
 {
     EngravingItem* e = data.dropElement;
     switch (e->type()) {
+    case ElementType::ACCIDENTAL:
+        score()->changeAccidental(toAccidental(e)->accidentalType());
```

Comment:
I think you need to change this to <code>score()->changeAccidental(<b>note()</b>, toAccidental(e)->accidentalType());</code> to fix the issue you asked about on Discord 🙂